nelmio/api-doc-bundle/2.13/etc/routing/nelmio_api_doc_routing.yaml

```diff
@@ -0,0 +1,3 @@
+NelmioApiDocBundle:
```

Comment:
The name should be `nelmio_api_doc`. See the web_profiler: https://github.com/symfony/recipes/blob/master/symfony/web-profiler-bundle/3.3/etc/routing/dev/web_profiler.yaml